pkgs/os-specific/linux/iputils/timespec.patch

```diff
@@ -0,0 +1,32 @@
+From b93f74990440a9fdde0238a262839c4f87b4a449 Mon Sep 17 00:00:00 2001
+From: Will Dietz <w@wdtz.org>
+Date: Sun, 24 Mar 2019 13:40:16 -0500
+Subject: [PATCH] tracepath: include fix
```

Comment:
wat?

Reply:
Fixes includes so this builds with musl, they regressed on this (looks like style/auto-format tool is responsible?) -- on the other issue I link to the upstream issue I opened about this.